packages/db/index.js

```diff
@@ -278,7 +278,7 @@ export class DBClient {
       .single()
 
     if (error) {
-      throw new DBError(error)
+      return undefined
```

Comment:
i think here it should catch the 206 status code or the `!data` not all others errors